datalad/distributed/create_sibling_ria.py

```diff
@@ -674,6 +678,17 @@ def _create_sibling_ria(
     else:
         gr = GitRepo(repo_path, create=True, bare=True,
                      shared=shared if shared else None)
+        if exists and existing == 'reconfigure':
+            # if the repo exists at the given path, the GitRepo would not
+            # (re)-run git init, and just return an instance of GitRepo;
+            # skip & error have been handled at this point
+            gr.init(
+                sanity_checks=False,
+                init_options=[
+                    "--bare",
+                    "--shared={}".format(shared) if shared else "",
```

Comment:
f-string can simply be used
```suggestion
                    f"--shared={shared}" if shared else "",
```

Reply:
Thanks, applying suggestion from below which also covers this bit.